src/test/java/programminglife/controller/BookmarkControllerTest.java

```diff
@@ -0,0 +1,65 @@
+package programminglife.controller;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import programminglife.model.Bookmark;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+/**
+ * Created by marti_000 on 16-5-2017.
+ */
+public class BookmarkControllerTest {
+    private Bookmark bookmark;
+    private String testPath;
+
+    @Before
+    public void setup() {
+        bookmark = new Bookmark( 2, 6,"testbm", "testing");
+        testPath = BookmarkControllerTest.class.getResource("/bookmarkTest.xml").getPath();
+    }
+
+    @After
+    public void cleanup() {
```

Comment:
Uses the tested class methods to cleanup. Will just delete the file in the future.